bin/webpack.js

```diff
@@ -292,6 +291,24 @@ function processOptions(options) {
 		}));
 	}
 
+	var lastHash = null;
+	var lastErrors = [];
+	var lastWarnings = [];
+
+	var arrayEquals = require("../lib/arrayEquals");
+
+	function shouldReprintStats(stats, statsJson) {
+		var reprint = stats.hash !== lastHash ||
+			!arrayEquals(lastErrors, statsJson.errors) ||
+			!arrayEquals(lastWarnings, statsJson.warnings);
+		if(reprint) {
+			lastHash = stats.hash;
+			lastErrors = statsJson.errors;
+			lastWarnings = statsJson.warnings;
```

Comment:
👍 updated, also added a test for `arrayEquals`. The CI seems to be broken though.